### DISCUSSION TITLE
August update: Change in training

### DISCUSSION THREAD

**Discussion by emelypi** [open-mmlab/mmyolo#926] [Q&A]
Hi there,

I pulled the recent version of mmYolo (August update). Since then, I am unable to recreate my experiments. I have set a seed for randomness in my config file, run the same config file with the same dataset and same pre-trained weights. With mmYolo pre-August, I achieve the same mAP value after 300 epochs. With the current version, my mAP value is considerable lower.

What causes the change in training?
